gcc/rust/checks/errors/borrowck/bir-design-notes.md

```diff
@@ -0,0 +1,188 @@
+# Borrow-checker IR (BIR) design notes
```

Comment:
should we have all these doc files in a `doc/` subdir?

Reply:
I am fine with either variant. If `docs/` is more consistent, we can go with that.